base/interactiveutil.jl

```diff
@@ -70,9 +70,17 @@ edit(file, line::Integer) = error("could not find source file for function")
 
 # terminal pager
 
-function less(file::AbstractString, line::Integer)
-    pager = get(ENV, "PAGER", "less")
-    run(`$pager +$(line)g $file`)
+if is_windows()
+    function less(file::AbstractString, line::Integer)
+        pager = get(ENV, "PAGER", "more")
+        g = pager == "more" ? "" : "g"
+        run(Cmd(`$pager +$(line)$(g) \"$file\"`, windows_verbatim = true))
```

Comment:
The escaping won't work without, I think. See below for a (possibly) nicer fix though.